knowledge-base/grid-scroll-specific-row.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Scroll to a specific row in the Kendo UI for Vue Native Grid
```

Comment:
Capitalize the title https://docs.telerik.com/style-guide/titles-and-headings#capitalization